apps/pre-award/copilot/environments/overrides/cfn.patches.yml

```diff
@@ -0,0 +1,22 @@
+# Delete the task role resource
+# - op: remove
+#   path: /Resources/TaskRole
+
+# Add a service connect alias
+# - op: add
+#   path: /Resources/Service/Properties/ServiceConnectConfiguration/Services/0/ClientAliases/-
+#   value:
+#     Port: !Ref TargetPort
+#     DnsName: yamlpatchiscool
+
+# Replace the task role in the task definition
+# - op: replace
+#   path: /Resources/TaskDefinition/Properties/TaskRoleArn
+#   value: arn:aws:iam::123456789012:role/MyTaskRole
+
+# TODO: Uncomment this to enable domain redirects
```

Comment:
Note this PR only deploys the function code, but it won't be enabled until a follow-up PR uncomments this block.

I haven't tested if it's possible to restrict this to one environment, so at the moment would need to only be enabled once all other changes have been merged.